src/object_ref.jl

```diff
@@ -1,14 +1,10 @@
 mutable struct ObjectRef
     oid_hex::String
-    owner_address_json::Union{String,Nothing}
+    owner_address::ray_jll.Address
```

Comment:
how does this interact with `deepcopy` of an `ObjectRef`?  when I tried this before one thing I ran into was that `deepcopy` doesn't actually copy C++-managed things, so when you finalize one deepcopy the other now is pointing to freed memory...

Reply:
I think the test case for this would be something like
- construct ObjectRef with non-nil owner address
- deepcopy it
- finalize one copy
- do something with the remaining copy that involves accessing the address field (even just serializing might be enough there...)

Reply:
Will do. I'll mention that in #204 things are refactored a bit so a `deepcopy` of a `ObjectRef` would no longer require copying the `Address`

Reply:
I created the test case you asked for and didn't find any problems with using `deepcopy`.